test/widgets/content_test.dart

```diff
@@ -328,6 +328,95 @@ void main() {
     });
   });
 
+  group("MessageEmbedVideo", () {
+    Future<void> prepareContent(WidgetTester tester, String html) async {
+      addTearDown(testBinding.reset);
+      await testBinding.globalStore.add(eg.selfAccount, eg.initialSnapshot());
+      prepareBoringImageHttpClient();
+
+      await tester.pumpWidget(GlobalStoreWidget(child: MaterialApp(
+        home: PerAccountStoreWidget(accountId: eg.selfAccount.id,
+          child: MessageContent(
+            message: eg.streamMessage(content: html),
+            content: parseContent(html))))));
+      await tester.pump(); // global store
+      await tester.pump(); // per-account store
+      debugNetworkImageHttpClientProvider = null;
+    }
+
+    testWidgets('video preview for youtube embed', (tester) async {
+      const example = ContentExample.videoEmbedYoutube;
+      await prepareContent(tester, example.html);
+
+      final expectedVideo = example.expectedNodes[1] as EmbedVideoNode;
+      final expectedResolvedPreviewUrl = eg
+        .store()
+        .tryResolveUrl(expectedVideo.previewImageSrcUrl)!;
```

Comment:
nit:
```suggestion
      final expectedResolvedPreviewUrl = eg.store()
        .tryResolveUrl(expectedVideo.previewImageSrcUrl)!;
```

("eg" alone means very little, so keep it together with the name of what we're pulling off it)